frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortletUtils.java

```diff
@@ -288,6 +299,13 @@ public static Set<Attachment> updateAttachmentsFromRequest(PortletRequest reques
     public static Release cloneRelease(String emailFromRequest, Release release) {
 
         Release newRelease = release.deepCopy();
+        final Map<String, String> extIdHmap = release.getExternalIds();
+        for (Entry<String, String> names : extIdHmap.entrySet()) {
+            if (names.getKey() != null && names.getKey().equals(EXTERNALID_BLOCKLIST_CLONING_RELEASE)) {
+                extIdHmap.remove(names.getKey());
```

Comment:
There is a possibility of `ConcurrentModificationException` here, so to avoid please consider to use `ConcurrentHashMap`